readme.md

```diff
@@ -41,8 +47,15 @@ class Application extends BaseApplication
         ]);
 
         // Load the authenticators, you want session first
-        $service->loadAuthenticator('Authentication.Session');
-        $service->loadAuthenticator('Authentication.Form');
+        $service->loadAuthenticator('Authentication.Session', [
+            'sessionKey' => 'Auth'
+        ]);
+        $service->loadAuthenticator('Authentication.Form', [
+            'fields' => [
+                'username' => 'email',
```

Comment:
It also makes all the examples in the readme consistant.

Perhaps
```php
'fields' => [
    'username' => 'email', // defaults to 'username'
    'password' => 'pswd', // defaults to 'password'
]
```

Then it can show both, or is that too busy?